### ISSUE TITLE
no module named eval_offline

### ISSUE THREAD

**Issue by Psycho-9** [HCIILAB/LAST#3]
when i python test.py, the following error happened:
ModuleNotFoundError: No module named 'eval_offline'

I wonder 'eval_offline' a package or a .py in your project ?

**Comment by W-Geong** [comment]
Indeed, this file is missing, but in reality, the result has already been calculated during inference and there is no need to call offline_ eval module. You can just ignore it. I will update a version that can be directly inferred in the near future. You can try to complete this project based on the code of BTTR and CoMER.
